cogs/stars.py

```diff
@@ -1,30 +1,32 @@
-import aiohttp
+from aiohttp import ClientSession
+from nextcord import TextChannel
 from nextcord.ext import commands
 from nextcord.ext.tasks import loop
 
+STARS_CHANNEL_ID: int = 884441198520045570
 
 class GitHubStars(commands.Cog):
     """Run a task loop to update a channel with the current stars for nextcord/nextcord and nextcord/nextcord-v3"""
 
-    STARS_CHANNEL_ID = 884441198520045570
-
     def __init__(self, bot: commands.Bot):
-        self.__bot = bot
+        self.bot: commands.Bot = bot
         self.update_stars.start()
 
+
     async def get_stars(self, repo: str) -> int:
         """Get number of GitHub stars for a given repo in the form 'owner/repository'"""
-        async with aiohttp.ClientSession() as session:
+        async with ClientSession() as session:
             async with session.get(f"https://api.github.com/repos/{repo}") as resp:
                 data = await resp.json()
```

Comment:
If your typehinting everything why not this?

Reply:
May have forgotten that one, oops.